src/lib.rs

```diff
@@ -378,18 +381,11 @@ impl Window {
         }
     }
 
-    pub fn main_loop(self, on_draw: impl FnMut(&mut imgui::Ui) -> ()) -> ! {
-        let on_draw = Box::new(on_draw);
-
-        // Allocate `clsoure` on the heap and erase the lifetime bound.
-        // This is safe because we will never leave this function (alive)
-        // The same applies for closure in on_init
-        let closure: Box<dyn FnMut(&mut imgui::Ui)> = Box::new(on_draw);
-        let closure: Box<dyn FnMut(&mut imgui::Ui) + 'static> =
-            unsafe { std::mem::transmute(closure) };
+    pub fn main_loop(self, on_draw: impl FnMut(&mut imgui::Ui) -> () + 'static) -> ! {
```

Comment:
Oh, I feel that I should not add this `'static` to this diff (as much as I would like to..)

examples are not building with this `'static` :(